sdcm/tester.py

```diff
@@ -1725,13 +1727,12 @@ def send_email(self):
         Child class should implement the method get_mail_data
         which return the dict with 2 required fields:
             email_template, email_subject
-        and set the self.email_reporter to appropriate reporter object
         """
         send_email = self.params.get('send_email', default=False)
         if send_email:
             try:
-                email_data = self.get_email_data()
                 if self.email_reporter:
```

Comment:
on line 191 email reported is set, this if is always true...

Reply:
Not always, only for Gemini/Longevity testis. There is used fabric function which return GeminiReporter or Longevity reporter based on child Tester classname, or None.
So if the Performance tests will be run the email_reporter will be None